lib/elixir_tic_tac_toe_basic.ex

```diff
@@ -23,7 +23,10 @@ defmodule ElixirTicTacToeBasic do
 
     is_game_over = outcome_checker.is_game_over(config)
 
-    if is_game_over, do: config, else: play_game(config)
+    case is_game_over do
+      false -> play_game(config)
+      true -> config
```

Comment:
**question**: Is this actually tail-recursive? 🤔

I know that I said that using a `case` at the end would be in a pairing session earlier this week, but I think I was wrong about that. Tail call optimization happens when that **last thing** in the function body is a recursive call. Do we know for sure?